backend/api/Controllers/UserController.cs

```diff
@@ -90,15 +83,15 @@ public IActionResult AddAccessRequest([FromBody] Model.AccessRequestModel access
         {
             if (accessRequestModel == null || accessRequestModel.Agencies == null || accessRequestModel.Roles == null)
             {
-                return BadRequest("Invalid access request specified");
+                throw new BadRequestException("Invalid access request specified");
```

Comment:
We want our middleware to handle responses so we get consistent JSON messages.

Reply:
See my previous comment about leaving a comment for other devs who might need to modify/maintain this area?